mainpage.dox

```diff
@@ -26,11 +26,20 @@ The basic bricks of the libiio API are the iio_context, iio_device, iio_channel
 - A iio_device object may contain zero or more iio_channel objects. A iio_channel object is associated with only one iio_device.
 - A iio_device object may be associated with one iio_buffer object, and a iio_buffer object is associated with only one iio_device.
 
+@section scan_contexts Scanning for IIO contexts
+The first step when dealing with a collection of IIO devices (known as a context) is to find the context. This can be connected via usb, network, serial or local. Having these different connectivity options could prove to be problematic, but libiio abstracts the low level communications away, and allows you just to find contexts, and talk to devices without being interested in the low level aspects. Many find this convinent to develop applications and algorithms on a host, and quickly move to an embedded Linux system without having to change any code.
+
+To find what IIO contexts are available, the first things to is use the following:
```

Comment:
Should it be: "first thing to do is"? (no trailing 's' in 'things' and add a 'do')

Reply:
yeah, that was awkward - thanks for reviewing - fixed.